blocks/library/text/index.js

```diff
@@ -31,13 +31,29 @@ registerBlockType( 'core/text', {
 		content: query( 'p', children() ),
 	},
 
+	transforms: {
+		from: [
+			{
+				type: 'raw',
+				matcher: ( node ) => (
+					node.nodeName === 'P' &&
+					// Do not allow embedded content.
+					! node.querySelector( 'audio, canvas, embed, iframe, img, math, object, svg, video' )
```

Comment:
How did we determine which node types to include here?

Reply:
A follow-up question: _Why_ did we exclude embedded content? Was this meant to allow other raw matchers for embedded content take precedence?